src/plugins/data/common/index_patterns/index_patterns/index_patterns.ts

```diff
@@ -368,20 +378,26 @@ export class IndexPatternsService {
       ? JSON.parse(runtimeFieldMap)
       : {};
 
+    const patternListActive = await this.apiClient.validatePatternListActive({
+      patternList: savedObject.attributes.patternList,
+    });
+
     return {
+      allowNoIndex,
```

Comment:
Apologies for the extra line changes, it is so much easier for me to see where updates to objects/types have been made when the properties are in alphabetical order. I did that a few times throughout this PR